package.json

```diff
@@ -159,6 +159,16 @@
         "command": "quarkusTools.deployToOpenShift.short",
         "title": "Deploy to OpenShift",
         "icon": "icons/openshift.png"
+      },
+      {
+        "command": "qute.validation.enabled.toggle.on",
+        "title": "Qute Validation Enabled",
+        "icon": "$(pass-filled)"
+      },
+      {
+        "command": "qute.validation.enabled.toggle.off",
+        "title": "Qute Validation Disabled",
+        "icon": "$(circle-large-outline)"
```

Comment:
I like ${eye-watch} and ${eye-closed}

![qute-validation-toggle](https://user-images.githubusercontent.com/148698/152215206-5575664d-b29e-49c7-8644-2f385ed9bc77.gif)